.circleci/config.yml

```diff
@@ -1635,6 +1669,13 @@ workflows:
       - t_win_soltest: *workflow_win
 
       # Bytecode comparison:
+      # Note: b_bytecode_ubu_static is required because b_ubu_static and b_ubu
+      # are currently built on different Ubuntu base images.
+      # It can be safely removed once we move both to the same Ubuntu version.
+      - b_bytecode_ubu_static:
```

Comment:
Can you put it on the job definition instead?

Reply:
Definitely.